CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 ## Latest
+* Update Roslyn version to 4.9.0-1.23502.8 (PR: [#6447](https://github.com/dotnet/vscode-csharp/pull/6447))
+  * Update Razor project configuration file name (PR: [#70156](https://github.com/dotnet/roslyn/pull/70156))
+
+## 2.5.?
```

Comment:
@dibarbet not sure what the normal process here is. Will it be 2.5.0? .1?

Reply:
it is most likely going to be 2.5.30 unless we have to re-spin for some reason (you can check the release branch build number).  You can use that, or ? and we'll update it when we actually release.

Reply:
I'll leave it then. More likely to notice it needs to be updated if its a question mark